skype_send_message.py

```diff
@@ -1,29 +1,26 @@
+import keyring
 import argparse
-import bcrypt
 from skpy import Skype
-import time
 parser = argparse.ArgumentParser()
 parent_parser = argparse.ArgumentParser(add_help=False)
+parent_parser.add_argument('--servicename', action="store")
 parent_parser.add_argument('--username', action="store")
-#ShDo = "" # variable for complete work, ShDo = "shutdown"
-outputfile = "contact"
-print ("Welcome to 'Skype send message'. ")
-username = str(input("Type your username of Skype: "))
-password = str(input("Type your password of Skype: "))
+parent_parser.add_argument('--password', action="store")
+print("Welcome to 'Skype send message'. ")
+username = input("Type your username of Skype: ")
```

Comment:
@idumenskyi considering we expect username coming from command line, could you please check if it was passed first, before asking for an input?